### DISCUSSION TITLE
How to fix pits on the bed?

### DISCUSSION THREAD

**Discussion by Latist1** [mriscoc/Ender3V2S1#961] [Q&A]
Hello guys. Im start using this firmware and its so cool. But I need your help. maybe someone know how to fix this?
I have tried with clips on different positions, without clips. And when I start print on the red spot filament not sticking  to plate, and on the blue almost scratching bed or scratching.  
![IMG_3552](https://github.com/mriscoc/Ender3V2S1/assets/131769862/76349513-cf1a-4512-ad35-433b40a1cb2e)

**Comment by Crazy-Charles** [comment]
Have you set and saved your Z offset?

If not, have a look here re: calibration   https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
